pyproject.toml

```diff
@@ -1,6 +1,260 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "fastentrypoints~=0.12"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "aiida-core"
+dynamic = ["version"]  # read from aiida/__init__.py
```

Comment:
We could define it statically instead and use a tool like [bumpver](https://pypi.org/project/bumpver/) to automate the consistent update of the version.

Reply:
But then you are back to having to enforce consistency checks, and use special tools like this. I would be against that

Reply:
Plus, even if you did decide to use bumpver, this would not be mutually exclusive.

Reply:
Indeed, adopting bumpver or similar would be a good way to simplify and standardize version updates regardless. I have started using this tool in more and more projects and found that the "single source of truth" for the version is less important than I thought.

Reply:
Are you happy then, that we leave in this dynamic version for now, then can revisit if you want to propose a PR to add bumpver?

Reply:
Yes.